packages/api/src/subsocial/index.ts

```diff
@@ -200,11 +348,33 @@ export class SubsocialApi implements ISubsocialApi {
   //------------------------------------------------
   // Profile spaces
 
+  /**
+   * Find and load an array of information about profile spaces from Subsocial blockchain and IPFS by a given array of
```

Comment:
```suggestion
   * Find and load an array of information about some profile spaces from Subsocial blockchain and IPFS by a given array of
```

Reply:
Not 100% sure about this one. Does this load information for ALL profile spaces? Or just one? Or can you get 5 at a time?

If you answer yes to more than one of those questions, my suggestion is correct.

Reply:
I'm pretty sure that each account can only set one profile space (confirming to @olehmell about this). 
So maybe we can rephrase it into `information about the profile space for each accountId in an array`? wdyt @Yung-Beef

Reply:
Find and load an array of information about the profile space of each accountId in the array from the Subsocial blockchain and IPFS.